go/pkg/retry/retry.go

```diff
@@ -60,6 +62,27 @@ const UnlimitedAttempts = Attempts(0)
 // Always always retries, regardless of error.
 func Always(error) bool { return true }
```

Comment:
Can you move `ShouldRetry` after `UnlimitedAttempts`? To group it closer with the retry check funcs.

Reply:
done